test/load/README.md

```diff
@@ -0,0 +1,26 @@
+# Load Testing
+
+Load tests aim to test the performance of the system under heavy load. Game server allocation is an example where multiple parallel operations should be tested.
+
+## Build and run test
+
+Prerequisites:
+- Docker.
+- A running k8s cluster.
+- Have kubeconfig file ready.
+
+Load tests are written using Locust. These tests are also integrated with Graphite and Grafana
+for storage and visualization of the results. 
+
+To run load tests using Locust on your local machine:
+
+```
+docker build -t locust-files .
```

Comment:
I would add the `kubectl proxy &` command

Reply:
Done.